05_stringer/selva/main_test.go

```diff
@@ -0,0 +1,24 @@
+package main
+
+import (
+	"bytes"
+	"strconv"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestMainOutput(t *testing.T) {
```

Comment:
Write functions `TestIPAddString` and use [table driven tests](https://github.com/golang/go/wiki/TableDrivenTests).

Reply:
Done.